### PR TITLE
Refactor atrule statement in unit-no-unknown

### PR DESCRIPTION
**Pull request by sylwia-lask**
> Which issue, if any, is this issue related to?

PR is related to discussion in #5500 - introduce isStandardAtRule. 

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

File: lib/rules/unit-no-unknown/index.js [hunk 1]
```diff
@@ -129,7 +130,7 @@ function rule(actual, options) {
 		}
 
 		root.walkAtRules((atRule) => {
-			if (!/^media$/i.test(atRule.name) && !atRule.variable) {
+			if (!/^media$/i.test(atRule.name) && !isStandardSyntaxAtRule(atRule)) {
```
Comment (ybiquitous) [inline lib/rules/unit-no-unknown/index.js:133]: [suggestion] Can we move the position of `/^media$/i` as below? I think it's more readable. 😃 

```js
		root.walkAtRules(/^media$/i, (atRule) => {
			if (!isStandardSyntaxAtRule(atRule)) {

```

**Review by ybiquitous — COMMENTED** [review]
@sylwia-lask Thanks for creating the PR! I've commented on a refactoring suggestion, so can you check it out?

File: lib/rules/unit-no-unknown/index.js (hunk 1 above)
Comment (jeddy3) [reply lib/rules/unit-no-unknown/index.js:133]: Yep, we should be using the `walk*` filter when available. I think most rules do, but it looks like this one must have slipped through at some point.

Comment (sylwia-lask) [reply lib/rules/unit-no-unknown/index.js]: @ybiquitous @jeddy3 thanks a lot guys for your review :)

**Review by ybiquitous — APPROVED** [review]
@sylwia-lask Thanks. LGTM! 😄

**Review by jeddy3 — APPROVED** [review]
@sylwia-lask Looks great, thank you!

**Pull request merged**